saladItem.js

```diff
@@ -1,3 +1,68 @@
 import React, { useReducer, useContext } from 'react';
 import PropTypes from 'prop-types';
 import { createUseStyles } from 'react-jss';
+
+import UserContext from '../User/User';
+import { SaladContext } from '../SaladMaker/SaladMaker';
+
+// Items for a list of Salad options for the Salad Maker Interface
+const useStyles = createUseStyles({
+        add: {
+          background: 'none',
+          border: 'none',
+          cursor: 'pointer',
+        },
+            favorite: {
+            fontSize: 20,
+            position: 'absolute',
+            top: 10,
+            right: 10,
+            },
+        image: {
+          fontSize: 80
+        },
+    wrapper: {
+        border: 'lightgrey solid 1px',
+        margin: 20,
+        padding: 25,
+        position: 'relative',
+        textAlign: 'center',
+        textTransform: 'capitalize',
+        width: 200,
+    }
+});
+
+var reducer = 0;
+const reducer = key => key + 1;
+// Provides a viewable list of Salad Items by Image and Name
+export default function SaladItem({ image, name }) {
+  const classes = useStyles();
+  const { setSalad } = useContext(SaladContext)
+  const user = useContext(UserContext);
+  const favorite = user.favorites.includes(name);
+  const [id, updateId] = useReducer(reducer, 0);
+  function update() { 
+      setSalad({ name, id: `${name}-${id}`})
+        updateId();
+    };
+
+  return(
+    <div className={classes.wrapper}>
+        <h3>
+          {name}
+        </h3>
+        <span className={classes.favorite} aria-label={favorite ? 'Favorite' : 'Not Favorite'}>
+          {favorite ? '😋' : ''}
+        </span>
+        <button className={classes.add} onClick={update} onTouchMove={}>
```

Comment:
- This is the button component that has the onClick event listener that fires off the update function.
- The onTouchMove event listener is empty and will not do anything when it fires off. Should consider removing it, or creating another event handler that will do something. Maybe swipe left for add, right for remove?